### DISCUSSION TITLE
My laser looks big and it causes my cutting and engraving to leave big mark. I tried to to turn the focus lens tight in and it's still big. My machine is an ORTUR Laser Master 20 watt.

### DISCUSSION THREAD

**Discussion by ikuemi** [arkypita/LaserGRBL#1129] [Q&A]

**Comment by arkypita** [comment]
For what I know there are 2 kind of 20W ORTUR modules: one that comes with adjustable focus ring, one that comes with a fixed focus ring.

The one with focus ring has an optimal distance at about 3-5 cm

The one with fixed focus has a very short focal distance, and comes with an alluminium block that can be use to measure that distance. It also have a ring but if you move the focus does not change.

![image](https://user-images.githubusercontent.com/8782035/94114686-f6cdbc00-fe48-11ea-950c-0453339fff18.png)

Post a picture of your modules to let me understand which kind you have.

PS: you can have a better support on ortur facebook community or ortur support forum:
https://www.facebook.com/groups/orturusergroup
https://ortur.tech/support/

**Comment by HeatSign5** [comment]
I’m sorry to hear that you’re having trouble with your [ORTUR Laser Master](url=https://www.heatsign.com/top-7-mini-laser-engraving-machine-for-metal/) 20-watt machine. If the laser beam is too big, it could be due to several reasons such as incorrect focus, a dirty lens, or a damaged lens. You can try cleaning the lens with a soft cloth and checking the focus again. If the problem persists, you may want to consider upgrading to a 20W laser head .
You can also take information from : https://www.heatsign.com/top-7-mini-laser-engraving-machine-for-metal/
